### PR TITLE
Update CEP Functions

### PR DESCRIPTION
**Pull request by jamesm-macrometa**
Hi @grainier-macrometa - can you please review the changes in the Functions doc for accuracy?  I've updated an extensive list of CEP function examples.  Please let me know if you have any questions or concerns.  Thanks!

### CODE REVIEW

File: docs/cep/reference/functions.md [hunk 1]
```diff
@@ -21,9 +21,9 @@ QUERY PARAMETERS
 EXAMPLE 1
 
 ```js
+    insert into alertStream
     select and(isFraud) as isFraudTransaction
-    from cscStream#window.lengthBatch(10)
-    insert into alertStream;
+    from cscStream#window.lengthBatch(10);
```
Comment (grainier-macrometa) [inline docs/cep/reference/functions.md:26]: @jamesm-macrometa - We have to change all `#window.xxxxx()` into QLv2 format.

https://docs.google.com/document/d/1RhZLZ2ab4vq_0ebMZuN1Mdpp8joUriWmyY0DRfrkilM/edit?disco=AAAAWtzue6U

Comment (jamesm-macrometa) [reply docs/cep/reference/functions.md:26]: Thanks @grainier-macrometa - does everything else look OK?

Do these need to be changed?

#window.frequent(...)

#window.unique:(...)

I guessed and went with "WINDOW FREQUENT()" and "WINDOW UNIQUE()

Comment (elof) [reply docs/cep/reference/functions.md]: @jamesm-macrometa are we all set here?

Comment (jamesm-macrometa) [reply docs/cep/reference/functions.md]: @elof Once @grainier-macrometa has confirmed we're good to go.

Nudge @grainier-macrometa for above questions 🙂

Comment (grainier-macrometa) [reply docs/cep/reference/functions.md]: @jamesm-macrometa - Frequent will be `WINDOW FREQUENT()`. However, it's deprecated now.  

As for the `unique:function`, it's `WINDOW UNIQUE:FUNCTION()`... Consider `unique` as a namespace (i.e `namespace:function`).

**Review by grainier-macrometa — APPROVED** [review]

**Comment by jamesm-macrometa** [comment]
Thanks @grainier-macrometa !

@elof This is good to go.

**Pull request merged**